modelci/hub/manager.py

```diff
@@ -195,21 +193,22 @@ def generate_model_family(
         generated_dir_list.append(tfs_dir.with_suffix('.zip'))
 
         # to TRT
-        converter.convert(net, 'tfs', 'trt', tf_path=tfs_dir, save_path=trt_dir, inputs=inputs, outputs=outputs, max_batch_size=32)
+        converter.convert(net, 'tfs', 'trt', tf_path=tfs_dir, save_path=trt_dir, inputs=inputs, outputs=outputs,
+                          max_batch_size=32)
         generated_dir_list.append(trt_dir.with_suffix('.zip'))
 
     return generated_dir_list
 
 
-def get_remote_model_weight(model: ModelBO):
+def get_remote_model_weight(model: MLModel):
```

Comment:
Is it better to put all local cache management to a separate file, e.g., `cache_manager`?